medusa/cache.py

```diff
@@ -79,6 +197,14 @@ def configure(cache_dir):
                           arguments={'filename': os.path.join(cache_dir, 'anidb.dbm'),
                                      'lock_factory': MutexLock})
 
+    # epsisode parsing (knowit / guessit) cache
+    episode_metadata.configure(
+        'dogpile.cache.dbm',
+        expiration_time=timedelta(days=365),
```

Comment:
maybe 365 days is a little much

Reply:
Any idea how long this would take on a large library? If it's not too long maybe 7 days if it's really slow maybe 90 days?

I could see this easily pegging my drives to 100% IO which would cause Plex, etc. to freeze and I have users 24/7.

Reply:
Depends how many episodes your scanning. Right now we aren't t using the metadata call.

But if you limit it to the postprocessing + dogpile storing, it shouldn't cause any load. Not on the biggest dbs.

Reply:
Wait the expiration time, is about how long to store episode metadata in dogpile